service/dash.go

```diff
@@ -71,7 +83,37 @@ func generateCappedBitrateAudio(targetFile string, outputFile string, fileType c
 }
 
 func generateCappedBitrateVideo(targetFile string, outputFile string, fileType constants.FILE_TYPE, wg *sync.WaitGroup) {
-	ffmpeg.Input(targetFile).
+
+	ffmpeg.Input(targetFile).Output(outputFile, ffmpeg.KwArgs{
```

Comment:
you can add the filter in this line itself
`ffmpeg.Input().Filter().Output()` like this

can you check if there is a way to make this optional

Reply:
@abhishekraj272 
ffmpeg.Input().Filter().Output() 
This won't work as .Input() returns ffmpeg.Stream*
and the Filter we want to use is ffmpeg.Filter